packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -605,6 +606,20 @@ export default class Meetings extends WebexPlugin {
       });
     }
 
+    /**
+     * Fetch user prefered webex site information
+     * This also have other infomation about the user
+     * @returns {Promise}
+     * @private
+     * @memberof Meetings
+     */
+    fetchUserPreferedWebexSite() {
+      return this.request.fetchLoginUserInfomation().then((res) => {
+        this.preferedWebexSite = MeetingsUtil.parsePreferedWebexSite(res.userPreferences);
```

Comment:
`parsePreferedWebexSite` name seems wrong as the function is parsing user preferences so should be named `parseUserPreferences`, shouldn't it?

Reply:
done